retro-lcd/usr/share/asteroid-launcher/watchfaces/retro-lcd.qml

```diff
@@ -38,7 +43,6 @@ Item {
             var ourDay = Qt.formatDate(wallClock.time, "dd").substr(0, 2).replace(/^0/, '');
 
             if (ourMins < 10) ourMins = "0" + ourMins;
-            if (ourSecs < 10) ourSecs = "0" + ourSecs;
```

Comment:
This line sounds like it should still be useful?

Reply:
indeed it is, that was accidentally deleted when there were two canvases